Solutions/Lookout/Data Connectors/Lookout_API_FunctionApp.json

```diff
@@ -68,7 +68,7 @@
 			"description": ">**NOTE:** This connector uses Azure Functions to connect to the Lookout REST API to pull its events into Azure Sentinel. This might result in additional data ingestion costs. Check the [Azure Functions pricing page](https://azure.microsoft.com/pricing/details/functions/) for details."
 		},
 		{
-			"description": ">**NOTE:** This data connector depends on a parser based on a Kusto Function to work as expected. [Follow these steps](https://aka.ms/sentinel-lookoutapi-parser) to create the Kusto functions alias, **LookoutEvents**"
+			"description": ">**NOTE:** This data connector depends on a parser based on a Kusto Function to work as expected. [Follow these steps](https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Solutions/Lookout/Parsers/LookoutEvents.txt) to create the Kusto functions alias, **LookoutEvents**"
```

Comment:
@rajendrakhabiyagate6  I have created short link for parser. https://aka.ms/sentinel-lookoutapi-parser
Please update description field value as follows.
` >**NOTE:** This data connector depends on a parser based on a Kusto Function to work as expected [**LookoutEvents**](https://aka.ms/sentinel-lookoutapi-parser) which is deployed with the Azure Sentinel Solution.`